tsdb/index/index_test.go

```diff
@@ -733,28 +619,17 @@ func TestChunksTimeOrdering(t *testing.T) {
 }
 
 func TestReader_PostingsForLabelMatchingHonorsContextCancel(t *testing.T) {
-	dir := t.TempDir()
-
-	idx, err := NewWriter(context.Background(), filepath.Join(dir, "index"))
-	require.NoError(t, err)
-
-	seriesCount := 1000
-	for i := 1; i <= seriesCount; i++ {
-		require.NoError(t, idx.AddSymbol(fmt.Sprintf("%4d", i)))
-	}
-	require.NoError(t, idx.AddSymbol("__name__"))
-
-	for i := 1; i <= seriesCount; i++ {
-		require.NoError(t, idx.AddSeries(storage.SeriesRef(i), labels.FromStrings("__name__", fmt.Sprintf("%4d", i)),
-			chunks.Meta{Ref: 1, MinTime: 0, MaxTime: 10},
-		))
+	const seriesCount = 1000
+	var input indexWriterSeriesSlice
+	for i := 1; i < seriesCount; i++ {
```

Comment:
I think the `<=` should be maintained:
```suggestion
	for i := 1; i <= seriesCount; i++ {
```

I think I might have made a mistake in my upstream PR, I'll try to rectify it in upstream.

Reply:
This feels potentially harmless? I would opt for leaving it as is and then just fixing it by syncing with upstream again.

Reply:
OK, should be harmless. I just merged my fix upstream.